packages/dev/materials/src/fire/fireMaterial.ts

```diff
@@ -21,6 +21,7 @@ import type { IAnimatable } from "core/Animations/animatable.interface";
 import "./fire.fragment";
 import "./fire.vertex";
 import { EffectFallbacks } from "core/Materials/effectFallbacks";
+import { addClipPlaneUniforms, bindClipPlane } from "core/Materials/clipPlaneMaterialHelper";

```

Comment:
Same comment for all materials of the material library.

Reply:
need core here as it is material library